docs/pre_build_command_example.md

```diff
@@ -1,4 +1,4 @@
-Example `build.sh` that uses a secondary `Dockerfile.build` to generate artifacts the main `Dockerfile`
+Example `build_binary.sh` that uses a secondary `Dockerfile.build` to generate artifacts the main `Dockerfile`
```

Comment:
I think this was right and used `build.sh` to build inside the docker container.